urld58fd5e7a9_longrm_weibo_robot_tgz-pJ8-weibo4j_org_json_TestJ8/src/com/ufgov/longrm/AccessTokenUtil.java

```diff
@@ -0,0 +1,66 @@
+package com.ufgov.longrm;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import weibo4j.http.AccessToken;
+
+public class AccessTokenUtil {
+
+  private static final String file = "accessToken.properties";
+
+  public static Map<String, String> getToken(String screenName) throws IOException {
+    Properties properties = new Properties();
+    FileInputStream inStream = new FileInputStream(file);
+    properties.load(inStream);
+
+    Map<String, String> tokenMap = new HashMap<String, String>();
+    tokenMap.put("USER_ID", properties.getProperty(screenName + ".USER_ID"));
+    tokenMap.put("TOKEN", properties.getProperty(screenName + ".TOKEN"));
+    tokenMap.put("TOKEN_SECRET", properties.getProperty(screenName + ".TOKEN_SECRET"));
+
+    inStream.close();
+    return tokenMap;
+  }
+
+  public static List<String> getScreenNameList() throws IOException {
+    Properties properties = new Properties();
+    FileInputStream inStream = new FileInputStream(file);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *inStream*. Execution paths that do not contain closure statements were detected. Ensure *inStream* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).